ramls/_schemas/kv_configuration.schema

```diff
@@ -30,6 +30,10 @@
     "userId": {
       "type": "string"
     },
+    "recordsSource": {
```

Comment:
Enum with values: "Source record storage", "Inventory" and "Source record storage and Inventory" can be more applicable here to protect API.